### ISSUE TITLE
dismissAllDialogs type missing

### ISSUE THREAD

**Issue by guillenotfound** [angrykoala/wendigo#474]
Where can it be used? Docs suggest `BrowserSettings`, but tests only cover `open` function AKA `OpenOptions`.

**Comment by angrykoala** [comment]
Yep, you are right, docs are wrong, this is an option to be used as:
```js
browser.open({
  dismissAllDialogs: true
})
```

I'll close this issue and open a new one to handle doc update
